File: berufsschule/11/AP/CPP/Hausaufgaben/3_OnlineBuch_Java/src/Kapitel.java

public class Kapitel 
{
	private int anzahlSeiten;
	private String name;
	
	Kapitel(int anzahlSeiten, String name)
	{
		this.anzahlSeiten = anzahlSeiten;
		this.name = name;
	}
	
	public int getAnzahlSeiten()
	{
		return anzahlSeiten;
	}
	public void setAnzahlSeiten(int anzahlSeiten)
	{
		this.anzahlSeiten = anzahlSeiten;
	}
	public String getName() 
	{
		return name;
	}
	public void setName(String name)
	{
		this.name = name;
	}
	public String toString()
	{
		String output =  name + " mit " + anzahlSeiten + " Seiten";
		return output;
	}
	
}

File: berufsschule/11/AP/CPP/Hausaufgaben/3_OnlineBuch/Kapitel.cpp
#include "Kapitel.h"

Kapitel::Kapitel()
{
	anzahlSeiten = 0;
	name = "kapitel";
}
Kapitel::~Kapitel()
{
}

int Kapitel::getAnzahlSeiten()
{
	return anzahlSeiten;
}
void Kapitel::setAnzahlSeiten(int zahl)
{
	anzahlSeiten = zahl;
}
char* Kapitel::getName()
{
	return name;
}
void Kapitel::setName(char *name)
{
	this->name = name;
}
